test/resolvers/project.js

```diff
@@ -124,4 +129,61 @@ describe('Project resolver', () => {
     });
   });
 
+  describe('create', () => {
+    it('creates a new project with an empty project version if called without a version param', () => {
+      const opts = {
+        action: 'create',
+        data: {
+          establishmentId,
+          licenceHolderId: profileId
+        }
+      };
+      return Promise.resolve()
+        .then(() => this.project(opts))
+        .then(() => this.models.Project.query())
+        .then(projects => {
+          assert(projects.length === 1, 'project not added');
+          return this.models.ProjectVersion.query().where({ projectId: projects[0].id })
+            .then(versions => {
+              assert(versions.length === 1, 'version not added');
+              assert.deepEqual(versions[0].data, null, 'empty version not added');
+            });
+        });
+    });
+
+    it('creates a new project and passes version data to a new version', () => {
+      const data = {
+        a: 1,
+        b: 2,
+        c: null,
+        d: false,
+        e: true,
+        title: 'This is the title'
+      };
+
+      const opts = {
+        action: 'create',
+        data: {
+          establishmentId,
+          licenceHolderId: profileId,
+          version: {
+            data
+          }
+        }
+      };
+      return Promise.resolve()
+        .then(() => this.project(opts))
+        .then(() => this.models.Project.query())
+        .then(projects => {
+          assert(projects.length === 1, 'project not added');
+          assert(projects[0].title, data.title, 'title not added to project');
```

Comment:
Not sure this assertion makes sense - it has too many arguments for a plain `assert`. Should it be `assert.equal`?